src/pages/trader-tools/pnl-for-margin/_pnl-margin-calculator.tsx

```diff
@@ -335,18 +331,16 @@ const PnlMarginCalculator = () => {
                                     setFieldValue('stopLossAmount', value)
                                 }
 
-                                const TakeProfitAmountInput = ({
-                                    field,
-                                }: {
-                                    field: React.ReactNode
-                                }) => (
+                                const TakeProfitAmountInput = () => (
                                     <Input
-                                        {...field}
                                         id="takeProfitAmount"
                                         type="text"
                                         label={localize('Take profit amount')}
                                         autoComplete="off"
-                                        error={touched.takeProfitAmount && errors.takeProfitAmount}
+                                        error={
+                                            touched.takeProfitAmount &&
+                                            errors.takeProfitAmount.toString()
```

Comment:
please use optional chaining to prevent unwanted errors

Reply:
The data is not dynamic here, I think we don't need that